src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -564,7 +564,7 @@ public final long write(ByteBuffer[] srcs) throws IOException {
      * @param   metaData
      *          If {@code true} then this method is required to force changes
      *          to both the file's content and metadata to be written to
-     *          storage; otherwise, it need only force content changes to be
+     *          storage; otherwise, it needs only force content changes to be
```

Comment:
(same as previous comment:  the suggested fix makes the English *less* correct)

Reply:
Reverted both in 9a9deee; thanks.